### ISSUE TITLE
The icon in the notification area should have a tooltip

### ISSUE THREAD

**Issue by PhiLhoSoft** [Code52/carnac#246]
After installing Carnac 2.3.3 (on Windows 10, 64-bit), it started to run immediately.

First, I am not fan of installers not asking where to install, and running without an warning…
But well, it works.

I wanted to see the program itself, its settings, etc. I supposed it is in the notification area, so I click there, and I see a new, purple icon. I hover over it, but I got not indication of what program it is. (A too common issue, I reckon…) If I right-click on it, I just get an Exit item menu, without indication of what I can exit either… I have to left click on it to finally have a confirmation it is really Carnac.

So this issue is to suggest to be a bit more user friendly with this icon…

Thanks for this software, BTW, seems to work fine!

**Comment by BinaryLoopCode** [comment]
Agreed, there most certainly should be both a tooltip and a "Settings" option in the right click menu!
At first I didn't think there were any settings since right clicking the tray icon only showed "Exit"
